src/lib/crypto/kimchi_bindings/stubs/kimchi_types.ml

```diff
@@ -18,58 +18,61 @@ type nonrec 'caml_f random_oracles =
   }
 [@@boxed]
 
+type nonrec 'evals point_evaluations = { zeta : 'evals; zeta_omega : 'evals }
+[@@boxed]
+
 type nonrec 'caml_f lookup_evaluations =
-  { sorted : 'caml_f array array
-  ; aggreg : 'caml_f array
-  ; table : 'caml_f array
-  ; runtime : 'caml_f array option
+  { sorted : 'caml_f array point_evaluations array
```

Comment:
with all the generics on proof systems it's hard to tell, but shouldn't this be `('camlf array) point_evaluations`?

Reply:
The type is right. It's an array (corresponding to the multiple sorted columns) of point evaluations, each of which is formed of an array (for chunking) of field elements.

Reply:
oh that's right!